pages/docs/open-source.mdx

```diff
@@ -15,9 +15,15 @@ Learn more about what's next for Langfuse on our [roadmap](/docs/roadmap).
 
 ## License
 
-Langfuse is licensed under an [MIT license](https://github.com/langfuse/langfuse/blob/main/LICENSE), with the exception of the `/ee` folder of the repository, if applicable.
+Langfuse is licensed under an MIT license, with the exception of the `/ee` and `/web/src/ee` folders of the repository. These contain features that are available within the Enterprise Edition of Langfuse and on Langfuse Cloud. See [License](https://github.com/langfuse/langfuse/blob/main/LICENSE) for more details.
 
-Currently, our focus is entirely on open source. However, we may consider commercializing premium features as part of an enterprise version in the future. For specific requirements, please contact us at enterprise@langfuse.com
+All Langfuse-maintained integrations and SDKs are fully open-source and licensed under the MIT license.
+
+## Enterprise Edition (EE) FAQ
+
+- What features are EE and what is MIT-licensed when self-hosting Langfuse? The core of Langfuse is all MIT licensed and open-source (tracing, integrations, public API, prompt management). Also, there are no restrictions on usage or deployment of these features when self-hosting Langfuse. The EE features are add-on features that are not required for the core functionality of Langfuse (e.g., model-based evaluation, llm playground, SSO enforcement, etc.). The documentation and codebase are clearly marked ("Where is this feature available?") to indicate which features are part of EE.
```

Comment:
What features are EE and what is MIT-licensed when self-hosting Langfuse? The core of Langfuse is MIT licensed and open-source (tracing, integrations, public API, prompt management). There are no restrictions on usage, modification or deployment of these features when self-hosting Langfuse. 
The EE features are commercially licensed add-on features that are not required for the core functionality of Langfuse (e.g., model-based evaluation, llm playground, SSO enforcement, etc.). 
The documentation and codebase are clearly marked ("Where is this feature available?") to indicate which features are MIT-licensed and which parts are EE (commercially licensed).